rubicon_ml/client/rubicon_json.py

```diff
@@ -214,6 +243,92 @@ def _rubicon_to_json(self, rubicon_objects):
 
         return json
 
+    def _fetch_parent_object(self, queried_object, id, return_type):
+        """
+        Returns the appropriate parent of the queried object to eventually append
+        to the return list of the query
+        """
+        if isinstance(queried_object, Project):
```

Comment:
this will never be true, `match.value` won't ever be a rubicon object

Reply:
Good catch, thank you, I'll just use the return_type as an indicator